src/agent/rustjail/src/container.rs

```diff
@@ -685,6 +685,7 @@ fn do_exec(logger: &Logger, path: &str, args: &[String], env: &[String]) -> Resu
         let v: Vec<&str> = e.split("=").collect();
         if v.len() != 2 {
             info!(logger, "incorrect env config!");
+            continue
```

Comment:
Thanks for raising @liubin and good catch! :smile:

But although this makes the code safer, I think it's still incorrect. Consider something like:

```sh
$ export foo=bar=baz=wibble
echo "$foo"
bar=baz=wibble
```

I think the code should be checking `if v.len() < 2`.